samples/charts/data-chart/radial-series/Pages/DataChartRadialSeries.razor

```diff
@@ -0,0 +1,193 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+
+    <div class="igOptions-horizontal">
+        <span class="igOptions-label">Type of Polar Series: </span>
+        <select value=@SeriesType
+                @onchange="OnSeriesTypeChanged">
+            <option>Area</option>
+            <option>Column</option>
+            <option>Pie</option>
+            <option>Line</option>
+        </select>
+
+        <span class="igLegend-title">Legend: </span>
+        <div class="igLegend">
+            <Legend @ref="LegendRef" Orientation="LegendOrientation.Horizontal" />
+        </div>
+    </div>
+
+    <div class="igComponent">
+        @if (DataSource != null)
+        {
```

Comment:
there should be 4 @if (SeriesType == "RadialAreaSeries" ) conditions each with one type of series (see StackedSeries sample):
RadialAreaSeries, RadialColumnSeries, RadialLineSeries, RadialPieSeries,